package.json

```diff
@@ -111,7 +111,8 @@
     "start": "concurrently --names server,frontend \"npm run start:server\" \"cross-env GATSBY_BASE_URL=http://localhost:8080 gatsby develop --port 3000\"",
     "e2e": "start-server-and-test start http://localhost:3000 test:e2e",
     "e2e-on-build": "cross-env CYPRESS_baseUrl=http://localhost:8080 start-server-and-test start:server:e2e-on-build http://localhost:8080 test:e2e",
-    "badge": "cross-env NODE_CONFIG_ENV=test TRACE_SERVICES=true node scripts/badge-cli.js"
+    "badge": "cross-env NODE_CONFIG_ENV=test TRACE_SERVICES=true node scripts/badge-cli.js",
+    "build-docs": "rm -rf api-docs/ && jsdoc --pedantic -c ./jsdoc.json . && mkdir -p ./api-docs/frontend/images/ && cp ./frontend/images/logo.svg ./api-docs/frontend/images/logo.svg"
```

Comment:
Unfortunately this script isn't very cross-platform :( There's a few couple of things going on here:

1. JSdoc doesn't clear the output dir to start with by default and it doesn't seem to have a `--clear` option or anything, but if you move files about, you get stale docs left around so it _is_ useful to clear the output dir first.
2. I've set `"readme": "README.md",` in the JSDoc config file so that the home page of the docs has something on it. I'm copying this image so that the top image on the home page isn't broken. Other solutions here might be to host the image somewhere and link directly to the image with absolute path (instead of a relative path), or we could write a bespoke markdown page to use as the documentation index instead of using the repo README, which would probably be better in the long-run.

Reply:
To clean the output cross-platformly, you can use `rimraf` like in `coverage:clean`.

For the logo, I think you could use this link:

https://raw.githubusercontent.com/badges/shields/master/frontend/images/logo.svg?sanitize=true

(see https://stackoverflow.com/a/16462143/893113)

Reply:
> cross-platformly

perfectly cromulent word ;)


> https://stackoverflow.com/a/16462143/893113

Useful - I didn't know about `?sanitize=true`